cypress/support/popover.po.ts

```diff
@@ -5,7 +5,77 @@ export class PopoverPo extends BaseComponent {
   pageTitle = 'Popover';
   ghLinkToComponent = 'https://github.com/valor-software/ngx-bootstrap/tree/development/src/popover';
 
+  togglerPopover = 'button';
+  containerPopover = 'popover-container';
+  body = 'body';
+
   exampleDemosArr = {
-    basic: 'demo-popover-basic'
+    basic: 'demo-popover-basic',
+    placement: 'demo-popover-placement',
+    dismiss: 'demo-popover-dismiss',
+    dynamic: 'demo-popover-dynamic',
+    customContent: 'demo-popover-custom-content',
+    dynamicHtml: 'demo-popover-dynamic-html',
+    appendToBody: 'demo-popover-container',
+    visibilityEvents: 'demo-popover-events',
+    eventTextBlock: 'pre.card.card-block.card-header.mb-3',
```

Comment:
1. bad selector, need to find more stable
2. it should be not in the exampleDemosArr